app/controllers/v0/addresses_controller.rb

```diff
@@ -41,26 +41,14 @@ def states
              serializer: StatesSerializer
     end
 
-    def rds_countries
-      response = strategy.cache_or_service(:countries) { rds_service.get_countries }
-      render json: response,
-             serializer: CountriesSerializer
-    end
-
-    def rds_states
-      response = strategy.cache_or_service(:states) { rds_service.get_states }
-      render json: response,
-             serializer: StatesSerializer
-    end
-
     private
 
     def service
-      EVSS::PCIUAddress::Service.new(@current_user)
-    end
-
-    def rds_service
-      EVSS::ReferenceData::Service.new(@current_user)
+      if Settings.evss&.reference_data_service&.enabled
```

Comment:
Did you consider moving this check outside the method so when the class is loaded the mode is determined instead of rechecking settings?  (I'm under the impression Settings can't be changed without restarting Rails) Might have to do some  stubbing to get tests to work though

Reply:
That's clever, I hadn't considered that. 

I think I'll save myself some extra commits to fix up tests and just leave it as is for now. But I'll keep this in mind.